docker/k8s/sgx-enclave-manager-kme-deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: avalon-enclave-manager-kme
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: avalon-kme
+  template:
+    metadata:
+      name: avalon-kme
+      labels:
+        app: avalon-kme
+    spec:
+      containers:
+      - image: avalon-sgx-enclave-manager-kme-dev:latest
+        name: avalon-kme
+        imagePullPolicy: Never
+        volumeMounts:
+        - mountPath: /dev/isgx
+          name: isgx
+        - mountPath: /var/run/aesmd/aesm.socket
+          name: aesmsocket
+        securityContext:
+          privileged: true
+        command: ["/bin/bash"]
+        args: ["-c", "enclave_manager --lmdb_url http://avalon-lmdb:9090 --bind http://0.0.0.0:1948"]
```

Comment:
We might need to modify this to pass WPE mrencalve value as arument. @rranjan3 is working on this

Reply:
https://github.com/hyperledger/avalon/pull/579/files#diff-0e475b27091475e3b287f8e0e5655a03R246